README.md

```diff
@@ -38,7 +38,7 @@ For other linux distributions, see [here](https://docs.djangoproject.com/en/2.2/
 
 The API docs rely on [drafter](https://github.com/apiaryio/drafter/) for parsing API Blueprint. This does not need to be installed manually on linux. On OSX this can be installed using
 ```
-brew install --HEAD https://raw.github.com/apiaryio/drafter/master/tools/homebrew/drafter.rb
+brew install drafter
```

Comment:
The existing install instruction here is probably wrong, but the "corrrection" probably is too.
django-apiblueprint-view needs specifically drafter v3.2.7
If you're not able to use the bundled shared object then you need to install that version. That command will install the latest version (v5.0.0) which will throw an error trying to render the API docs.
I've never used homebrew (or a mac) so I have no idea what the relevant brew incantation would be but it is probably.. not that.

Reply:
Thanks for flagging that Chris. Since this has an issue (#1400) I'm happy for it to be merged - I don't think it's _more_ broken... 
Since this PR is the precursor to Virginia doing some work on WDIV design I guess we'll see if building the docs locally is needed, and cross that bridge then.

Reply:
> Since this has an issue (#1400) 

Oh well done Chris-from-the-past ;)

> I guess we'll see if building the docs locally is needed

In order to drop dc_base_theme and replace it, it will be necessary to render the API docs so you probably do need to solve this one way or another.